scoring.js

```diff
@@ -1,24 +1,23 @@
-calculateScore = (player) => {
-    let result;
+const calculateScore = (player) => {
+  let result
 
-    if (player.position === 'QB') {
-        let passing = player.stats.passing;
-        let rushing = player.stats.rushing;
+  if (player.position === 'QB') {
```

Comment:
Your code can be consolidated more by using calculate passing, rushing, receiving and return functions.